source/slang/slang-ir-bit-field-accessors.cpp

```diff
@@ -0,0 +1,160 @@
+#include "slang-ir-bit-field-accessors.h"
+
+#include "slang-ir.h"
+#include "slang-ir-insts.h"
+
+namespace Slang
+{
+static IRInst* maybeUnwrapGeneric(IRInst* inst)
```

Comment:
There is already a `getResolvedInstForDecoration` function that unwrapps all generic/specialize insts and returns the inner thing being specialized.

Reply:
Ah I didn't see this before merging, will address tomorrow.

Reply:
No worries, it's a very minor issue.